### ISSUE TITLE
Icon

### ISSUE THREAD

**Issue by rumnat** [RestlessThinker/danger-jira#16]
Add possibility to set both icons please

**Comment by RestlessThinker** [comment]
Hi @rumnat, are you referring to the book and paperclip icons?

**Comment by rumnat** [comment]
Yes :)

**Comment by rumnat** [comment]
is it possible to fetch a summary along with the key name? Like **KEY-123 Update profile**

**Comment by lpenaflor** [comment]
@rumnat Are you suggesting that it matches the issue key from JIRA and updates the PR title for you?

**Comment by rumnat** [comment]
No, the way it looks. Now it's just **KEY-123**. Can it be **Key-123 Update profile**?
